styles/styles.scss

```diff
@@ -22,6 +22,8 @@
     @include yc-brand;
     @include yc-colors-social;
 
+    --yc-animation-multiplier: 1;
```

Comment:
Do we need to set it explicitly? We already have the same fallback for it.

Reply:
Of cause not, but we set default in fallback and user can change it by setting the var.

Reply:
Yes, that is how it should work

Reply:
and why are we setting the var?

Reply:
To debug animations in project, for example

Reply:
why are we setting it here?

Reply:
To see it dev-tools and adjust it if needed